src/hooks/GlobalUser.ts

```diff
@@ -1,34 +1,39 @@
 import { useEffect, useState } from "react";
-import store from "state";
+import store, { GlobalStore, GlobalStoreKey } from "state";
+
+const defKeys: GlobalStoreKey[] = [
```

Comment:
If you don't supply keys to `useGlobalUser`, you will receive updates when any of these keys change.